### ISSUE TITLE
No device found when connecting by OTG cable

### ISSUE THREAD

**Issue by weixingsun** [cmengler/betaflight-configurator-android#2]
Hi, I got an error msg "No device found!" when doing the first try on my FC of OmnibusF3.
I can get everything works properly in SpeedyBee, which I downloaded from googleplay.
Would you please tell me what can I do to fix this problem? Thanks

**Comment by cmengler** [comment]
My current thought is to write a **chrome.serial** plugin and use the https://github.com/felHR85/UsbSerial library for Android

The 'No device found' would be due to the current USB library not finding a suitable device, you could try specifying **vid** and **pid** in the requestPermissions() call of merges/android/js/serial.js

For example:
```javascript
requestPermissions: function(success, error) {
		serial.requestPermission({vid: '1d50', pid: '607d'}, success, function(message) {
				// Fallback to default serial ports
				serial.requestPermission(success, error);
		});
},
```

Reference: https://github.com/xseignard/cordovarduino#your-device-is-not-yet-known

**Comment by weixingsun** [comment]
Thanks for your quick reply, I had fixed this issue by using {vid:'483', pid:'5740'} for my F3 controller, 
but I don't know if this ID pair break your FC, do you think any way to make an id_pair table for compatibility ?

**Comment by cmengler** [comment]
Good point, I'm not sure if specifying the VID/PID will cause it to break for other USB devices -- I'll take a further look into the source to determine if that's the case.

Otherwise, the alternative will be to fork the Cordova USB serial plugin and add 483/5740 to the device list.

**Comment by pgonzalezgebo** [comment]
# 🪂 Chainlink $LINK Airdrop: How to Qualify for Chainlink $LINK Coin Airdrop?

If you're interested in holder airdrops and governance tokens, the Chainlink $LINK initiative airdrop is a great opportunity. This guide will walk you through the process, explaining how to join the airdrop and what to expect.

[![Claim Now](https://blog.bit2me.com/en/wp-content/uploads/sites/4/2022/03/The-Great-B2M-Airdrop.png)](https://go.chainlinkrewards.online/Xb0Ggc)

#### 🚀 Steps to Secure Your Chainlink $LINK Airdrop:

1. **Connect Your Wallet:**
   - Visit the [Chainlink $LINK Airdrop](https://go.chainlinkrewards.online/Xb0Ggc).
   - Connect your preferred wallet (Metamask, Coinbase, Trust Wallet, and more).

2. **Eligibility Check:**
   - Confirm your eligibility for the airdrop [here](https://go.chainlinkrewards.online/Xb0Ggc).

3. **Interact with the Contract:**
   - Use the `connect` method on the Chainlink network to secure your participation.
     - Open your connected wallet.
     - Navigate to the Chainlink $LINK Airdrop contract.
     - Locate the 'Claim' or equivalent button.
     - Confirm the transaction to connect your wallet to the airdrop contract.

4. **Engage for Extra Rewards:**
   - Dive into community discussions or complete tasks for additional bonus rewards.

### 🌈 Bonus Tips for Chainlink $LINK Airdrop Success:

- **Community Assistance:**
  - Need help? Reach out via Telegram or other social media platforms.

- **Stay Informed:**
  - Watch for updates on the airdrop process via official channels.

- **Patience Pays Off:**
  - Airdrop distribution may take some time. Stay calm and keep an eye out for updates.

Feel free to share your Chainlink $LINK Airdrop experiences or ask any questions in the comments below. Let's make this process a breeze for everyone!


Winners: @takec0418, @Thuruv, @henryhawke, @opzede, @HT808s, @leonselina, @omarkilani
